DependencyInjection/SonataMediaExtension.php

```diff
@@ -44,6 +44,27 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('extra.xml');
         $loader->load('form.xml');
         $loader->load('gaufrette.xml');
+
+        $defS3 = $container->getDefinition('sonata.media.adapter.service.s3');
```

Comment:
can we rename `$defS3`, this looks weird

Maybe `$amazonS3Definition` ??

/cc @greg0ire